source/using-ethereum-the-basics/ether.rst

```diff
@@ -5,27 +5,58 @@ Ether
 What is Ether?
 ================================================================================
 
+Ether is the base unit of currency used within Ethereum.  It is used to pay for
```

Comment:
Isn't Wei the base unit? Ether is also often referred to as _crypto fuel_ required to run the blockchain.